### ISSUE TITLE
OpenBMC GUI: Full hostname not displayed correctly in the BMC IP address

### ISSUE THREAD

**Issue by sivassrr** [openbmc/openbmc#2648]
@iftekharuli With full hostname not displayed correctly in BMC IP address
![image](https://user-images.githubusercontent.com/19144129/32961298-04d67b92-cbee-11e7-845f-75ce4d3abbe2.png)

**Comment by sivassrr** [comment]
May be fix can as follows under the app/common/dataService.js
```
            this.getServerId = function(){
              /*  return this.host.replace(/[^/D]+/m,"");  */
                 return this.host.replace(/^https?\:\/\//ig,"");
            }
```
It works.

**Comment by sivassrr** [comment]
![image](https://user-images.githubusercontent.com/19144129/33261238-3238d5c0-d388-11e7-8c08-287762aeba8a.png)

**Comment by sivassrr** [comment]
![image](https://user-images.githubusercontent.com/19144129/33261414-b563d4d6-d388-11e7-8e6f-2a3a06c34e3b.png)

**Comment by rfrandse** [comment]
<https://gerrit.openbmc-project.xyz/8693> Fix to get full BMC IP address / hostname
Resolves  #2648 OpenBMC GUI: Full hostname not displayed correctly in the BMC IP address
